### PR TITLE
SEO-178901-mvc-page-with-redirect

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Please review the redirect link changes made
Thanks.

Regards,
Edith.

### CODE REVIEW

File: aspnetmvc/Chart/3D-Chart.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ You can create a 3D Stacked Column Chart by setting the series **Type** as **Sta
 ![](3D-Chart_images/3D-Chart_img4.png)
 
 
-[Click](https://mvc.syncfusion.com/demos/web/chart/stackingcolumn1003d) here to view the 100% Stacked Column 3DChart online demo sample.
+[Click](https://ej2.syncfusion.com/aspnetmvc/ThreeDimensionalChart/StackedColumn#/bootstrap5) here to view the 100% Stacked Column 3DChart online demo sample.
```
Comment (Yvone-Atieno) [inline aspnetmvc/Chart/3D-Chart.md:114]: Check the updated link

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: aspnetmvc/Chart/3D-Chart.md (hunk 1 above)
Comment (Edithabich) [reply aspnetmvc/Chart/3D-Chart.md]: resolved

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Pull request merged**